packages/autorest.go/test/autorest/additionalpropsgroup/zz_models_serde.go

```diff
@@ -100,6 +100,8 @@ func (p *PetAPInProperties) UnmarshalJSON(data []byte) error {
 		case "status":
 			err = unpopulate(val, "Status", &p.Status)
 			delete(rawMsg, key)
+		default:
```

Comment:
I was surprised by this. I think we have a codegen bug (unrelated to this change).

Reply:
Opened https://github.com/Azure/autorest.go/issues/1069 to track this.